### PR TITLE
[build] Preparing 2.1.2 release

### PR DESCRIPTION
**Pull request by grgrzybek**

### CODE REVIEW

**Comment by rhuss** [comment]
Looks good to me. Thanks!

**Review by rhuss — APPROVED** [review]
thanks!

File: src/site/asciidoc/news.adoc [hunk 1]
```diff
@@ -15,6 +15,21 @@
...
 == News
 
+=== Jolokia 2.1.2 released
+
+[.news-date]
+2024-11-13
+
+Jolokia 2.1.2 is released and it is ready for more generic integration with other runtimes, especially with https://activemq.apache.org/components/artemis/[Apache Artemis,role=externalLink,window=_blank].
+
+Long existing `ServerDetector` abstraction was enhanced to let Jolokia know about actual ClassLoader used withing given runtime. For example, https://tomcat.apache.org/[Apache Tomcat] starts with just few JARs on system CLASSPATH and later the _actual_ ClassLoader is constructed using JARs from `${catalina.home}/lib`. Now Jolokia running as JVM agent has proper access to classes from this broader ClassLoader.
```
Comment (apupier) [inline src/site/asciidoc/news.adoc:25]: withing --> within

File: src/site/asciidoc/news.adoc [hunk 2]
```diff
@@ -15,6 +15,21 @@
...
+
+Additionally we've provided a little diagnostic check to detect (usually incorrect) setup when Jolokia is running twice in single JVM (for example as WAR and JVM agent).
```
Comment (apupier) [inline src/site/asciidoc/news.adoc:27]: Is there some documentation for it somewhere? Maybe we can link it?

File: src/site/asciidoc/news.adoc (hunk 1 above)
Comment (grgrzybek) [reply src/site/asciidoc/news.adoc:25]: darn - I always do that... Thanks for spotting!

File: src/site/asciidoc/news.adoc (hunk 2 above)
Comment (grgrzybek) [reply src/site/asciidoc/news.adoc:27]: I'll do it after the release - see #767

**Pull request merged**